Various small corrections (#1316)
carpentries/instructor-training: _episodes/01-welcome.md

Software Carpentry, Data Carpentry, and Library Carpentry
are official Lesson Programs of The Carpentries.

> Around them, The Carpentries forms a **global community of volunteer researchers, educators, and others** oriented around

Reviewer: ```suggestion
The Carpentries is a **global community of volunteer researchers, educators, and others** oriented around
```

The original sentence sort of had historical baggage that doesn't need to be here. We expand on this later, so I think an overall simpler sentence is more clear here.

Author: I adopted your suggestion. Thanks
